projects/plugins/jetpack/extensions/index.json

```diff
@@ -43,7 +43,7 @@
 		"post-publish-qr-post-panel",
 		"payment-buttons"
 	],
-	"beta": [ "amazon", "google-docs-embed", "recipe", "videopress-block" ],
+	"beta": [ "amazon", "google-docs-embed", "recipe", "jetpack/videopress" ],
```

Comment:
We usually do not need the `jetpack` prefix. Why do we need it here?

Reply:
this extension is going to conditionate whether the VideoPress block will be registered, or not. It raises up two things:
* Probably we're gonna change it to `videopress/video`
* I prefer to keep the namespace because gives more context to the feature, but I don't have a strong opinion, either.